source/type/type.cpp

```diff
@@ -154,6 +154,16 @@ bool HasMessages::has_publishable_message(const boost::posix_time::ptime& curren
     return false;
 }
 
+
+StopTime StopTime::clone() const{
+    StopTime ret{arrival_time, departure_time, stop_point};
+    ret.properties = properties;
+    ret.local_traffic_zone = local_traffic_zone;
+    ret.vehicle_journey = vehicle_journey;
```

Comment:
Please, put nullptr on vehicle_journey on a clone, as the stop time will be deconnected of the vj.